index.js

```diff
@@ -122,24 +118,50 @@ const plugin = (opts = {}) => {
         return;
       }
 
+      // Do not treat cloned rules already handled
+      if (
+        rule.selector.includes(conditionalNotSelector) ||
+        rule.selector.includes(containerBodySelector) ||
+        rule.selector.includes(conditionalSelector)
+      ) {
+        return;
+      }
+
       // Process rule if it needs conversion
       if (ruleProcessor.needsProcessing(rule)) {
         debugUtils.stats.rulesProcessed++;
         debugUtils.log(`Processing rule: ${rule.selector}`, rule);
 
         // Create container version with converted units
+        // should target [data-apos-refreshable-body]
         const containerRule = rule.clone({
           source: rule.source,
-          from: helpers.result.opts.from
+          from: helpers.result.opts.from,
+          selector: selectorHelper.addTargetsToSelectors(
+            rule.selector,
+            containerBodySelector
+          )
         });
-        containerRule.selector = `${conditionalSelector} ${rule.selector}`;
+
+        rule.selector = rule.selector.replace(
+          selectorHelper.bodyRegex,
+          conditionalNotSelector
+        );
+
         ruleProcessor.processDeclarations(containerRule, {
           isContainer: true,
           from: helpers.result.opts.from
         });
 
         // Add container rule after original
         rule.after('\n' + containerRule);
+      } else {
+        if (rule.selector.match(selectorHelper.bodyRegex)) {
+          rule.selector = selectorHelper.addTargetsToSelectors(
+            rule.selector,
+            [ conditionalNotSelector, containerBodySelector ]
+          );
+        }
```

Comment:
Process rules out of media queries, without units only if they have body selector